src/site/content/en/fast/signed-exchanges-webpackager/index.md

```diff
@@ -196,11 +196,36 @@ used with signed exchanges.
         you try to fetch pages from a different domain without updating
         `webpkgserver.toml`, the `webpkgserver` logs will show the error message
         `URL doesn't match the fetch targets`.
-    *   (Optional) To preload subresources, change the line `#PreloadCSS
-        = false` to `PreloadCSS = true`. In addition, change the line `#PreloadJS =
-        false` to `PreloadJS = true`. For more information about subresource
-        substitution, check out
-        [the explainer](https://github.com/WICG/webpackage/blob/master/explainers/signed-exchange-subresource-substitution.md).
+
+    **Optional**
```

Comment:
I think this is a good candidate for details component: https://web.dev/handbook/web-dev-components/#details